Makefile

```diff
@@ -5,6 +5,7 @@ default: build
 
 build:
 	go build -o bin/terraform-provider-tanzu-mission-control_v1.0.0
+	rm -rf ~/.terraform.d/plugins/vmware/dev/tanzu-mission-control/*
```

Comment:
the clean up steps should not be part of the build tag, please introduce another tag as cleanup and you can add it as part of it